docker-monolith/infra/ansible/roles/monolith/tasks/container_launcher.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Update package manager
+  ansible.builtin.apt:
+    update_cache: true
+    cache_valid_time: 300
+
+- name: Install python-pip
+  ansible.builtin.apt:
+    name: python3-pip
+    state: present
+
+- name: Install docker
+  ansible.builtin.pip:
+    name: docker
+    state: present
+
+- name: Launch a Docker container
+  ansible.builtin.docker_container:
+    name: reddit
+    image: fallgeratoor/otus-reddit:1.0
+    state: started
+    restart_policy: always
+    network_mode: bridge
+    ports:
+      - "9292:9292"
```

Comment:
In terraform you parametrize port but here not, so you should to parametrize `{{role_name_reddit_monolith_port}}:9292`